bin/convert-argv.js

```diff
@@ -15,6 +15,12 @@ module.exports = function(optimist, argv, convertOptions) {
 		process.exit(-1); // eslint-disable-line
 	}
 
+	// Version
+	if(argv.v || argv.version) {
+		console.log(require("../package.json").version);
+		process.exit(-1); // eslint-disable-line
```

Comment:
should the help section above this have an exit code of 0 also?

```
if(argv.help) {
		optimist.showHelp();
		process.exit(-1); // eslint-disable-line
}
```
I didn't write that code, but I was trying to be consistent with what was there already.  Should that be changed also?